api/apps/geoprocessing/test/integration/planning-unit-inclusion/world.ts

```diff
@@ -0,0 +1,133 @@
+export const createWorld = async (app: INestApplication) => {
+  const scenarioId = v4();
+  const puGeometryRepo: Repository<PlanningUnitsGeom> = app.get(
+    getRepositoryToken(PlanningUnitsGeom),
+  );
+  const scenarioPuDataRepo: Repository<ScenariosPlanningUnitGeoEntity> = app.get(
+    getRepositoryToken(ScenariosPlanningUnitGeoEntity),
+  );
+
+  const storedGeometries: string[] = [];
+  const geoToBeExcluded: string[] = [];
+  const geoToBeIncluded: string[] = [];
+  const geoToBeUntouched: string[] = [];
+
+  return {
+    scenarioId,
+    geoToBeExcluded: () => geoToBeExcluded.sort(sortUuid),
+    geoToBeIncluded: () => geoToBeIncluded.sort(sortUuid),
+    geoToBeUntouched: () => geoToBeUntouched.sort(sortUuid),
+    GivenPlanningUnitsExist: async (
+      planningUnits: AreaUnitSampleGeometry,
+    ): Promise<void> => {
+      const toInclude = await puGeometryFromGeoJson(
+        puGeometryRepo,
+        planningUnits.features.filter((f) => f.properties.shouldBeIncluded),
+      );
+      const toExclude = await puGeometryFromGeoJson(
+        puGeometryRepo,
+        planningUnits.features.filter((f) => f.properties.shouldBeExcluded),
+      );
+      const untouched = await puGeometryFromGeoJson(
+        puGeometryRepo,
+        planningUnits.features.filter(
+          (f) =>
+            !f.properties.shouldBeExcluded && !f.properties.shouldBeIncluded,
+        ),
+      );
+
+      storedGeometries.push(...toInclude, ...toExclude, ...untouched);
+      geoToBeExcluded.push(...toExclude);
+      geoToBeIncluded.push(...toInclude);
+      geoToBeUntouched.push(...untouched);
+
+      await scenarioPuDataRepo.save(
+        storedGeometries.map((id, index) =>
+          scenarioPuDataRepo.create({
+            puGeometryId: id,
+            scenarioId,
+            planningUnitMarxanId: index,
+          }),
+        ),
+      );
+    },
+    GetLockedInGeometries: async () =>
+      (
+        await scenarioPuDataRepo.find({
+          where: {
+            scenarioId,
+            lockStatus: LockStatus.LockedIn,
+          },
+        })
+      )
+        .map((entity) => entity.puGeometryId)
+        .sort(sortUuid),
+    GetLockedOutGeometries: async () =>
+      (
+        await scenarioPuDataRepo.find({
+          where: {
+            scenarioId,
+            lockStatus: LockStatus.LockedOut,
+          },
+        })
+      )
+        .map((entity) => entity.puGeometryId)
+        .sort(sortUuid),
+    GetUnstatedGeometries: async () =>
+      (
+        await scenarioPuDataRepo.find({
+          where: {
+            scenarioId,
+            lockStatus: LockStatus.Unstated,
+          },
+        })
+      )
+        .map((entity) => entity.puGeometryId)
+        .sort(sortUuid),
+    cleanup: async () => {
+      await puGeometryRepo.delete({
+        id: In(storedGeometries),
+      });
+      await scenarioPuDataRepo.delete({
+        scenarioId,
+      });
+    },
+  };
+};
+
+const puGeometryFromGeoJson = async (
```

Comment:
`puGeometryFromGeoJson` -> `insertPuGeometryFromGeoJson`

Reply:
thanks, fixed in https://github.com/Vizzuality/marxan-cloud/pull/258/commits/4ac31134f1a72bfa1f499577fd9f2aa2c1db2f6a